### DISCUSSION TITLE
Phone Auth

### DISCUSSION THREAD

**Discussion by irshadcholakkal** [pocketbase/pocketbase#4628] [Q&A]
Phone Auth is also needed

**Comment by aus-listed-cios** [comment]
Hate to reply with just links, but since your question was all of 5 words there's not much to go on. In any case, you can build this yourself if you're looking for magic link / OTP style auth. Also if you're looking for phone number verification.

Magic links and general custom auth: https://github.com/pocketbase/pocketbase/issues/119
Verification: https://github.com/pocketbase/pocketbase/discussions/350
Phone number auth: https://github.com/pocketbase/pocketbase/issues/279
